bin/grade_students.sh

```diff
@@ -837,44 +840,51 @@ while true; do
 	global_submission_time=""
 	global_gradeable_deadline=""
 	# call the helper function
+        echo "========================================================================"
 	grade_this_item $NEXT_DIRECTORY $NEXT_ITEM
 
+        ${SUBMITTY_INSTALL_DIR}/bin/grade_item.py ${NEXT_DIRECTORY} ${NEXT_TO_GRADE} ${ARGUMENT_UNTRUSTED_USER}
```

Comment:
Is this a potential problem that it's using both the old style and new style of graders at the same time?